src/Testing/MockClientStreamingCall.php

```diff
@@ -74,16 +73,18 @@ public function wait()
 
     /**
      * Save the request object, to be retrieved via getReceivedCalls()
-     * @param $request The request object
+     * @param \Google\Protobuf\Internal\Message $request The request object
```

Comment:
It seems like requests don't have to be this type (looking at the implementation)